tests/conftest.py

```diff
@@ -254,12 +254,16 @@ def wait_for_window_of(self, wmclass):
         return line[-1]
 
     def shutdown(self):
+        # first send SIGTERM to all processes, so they
+        # can shut down in parallel
         for client_proc in self.client_procs:
             client_proc.terminate()
-            client_proc.wait(2)
-
         self.hc_idle.terminate()
-        self.hc_idle.wait(2)
+
+        # and then wait for each of them to finish:
+        for client_proc in self.client_procs:
+            client_proc.wait(5)
+        self.hc_idle.wait(5)
```

Comment:
You might want to extract the 5 into a constant variable, so in the future we can just increase the value in one place instead of many. HLWM_WAIT_TIMEOUT or something?

Reply:
I've created a new global variable; and now the CI is triggered again :)

Reply:
Nice, thanks!